tests/test_categorical.py

```diff
@@ -1168,7 +1168,25 @@ def test_vs_catplot(self, long_df, wide_df, null_df, flat_series, kwargs):
         g = catplot(**kwargs, kind="box")
 
         assert_plots_equal(ax, g.ax)
-
+        
+        
+    @pytest.mark.parametrize("positions", [None, [1, 2, 3, 4]])
+    def test_reject_boxpositions(self, positions):
+        
+        kwargs = {
+            "x":[1, 2, 3, 4],
+            "y":[1, 1, 2, 2]
+        }
+            
+        try:
+            boxplot(**kwargs, positions=positions)
+            err = None
+        except ValueError as e:
+            err = e
+        
+        assert(err is not None)
```

Comment:
It would be slightly more idiomatic I think to use `pytest.raises` here, ideally with the `match` parameter to make sure we're getting the error we expect.